modules/tinymce/src/plugins/visualblocks/test/ts/browser/PreviewFormatTest.ts

```diff
@@ -1,45 +1,41 @@
-import { Assertions, Log, Pipeline, Step, Waiter } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader } from '@ephox/mcagar';
-import { Class, Css, SugarElement } from '@ephox/sugar';
+import { Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyDom, TinyHooks } from '@ephox/mcagar';
+import { Class, Css } from '@ephox/sugar';
+import { assert } from 'chai';
 
-import VisualBlocksPlugin from 'tinymce/plugins/visualblocks/Plugin';
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/visualblocks/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.visualblocks.PreviewFormatsTest', (success, failure) => {
-  Theme();
-  VisualBlocksPlugin();
+describe('browser.tinymce.plugins.visualblocks.PreviewFormatsTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'visualblocks',
+    toolbar: 'visualblocks',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin, Theme ]);
 
-  const sWaitForVisualBlocks = (editor) => {
-    return Waiter.sTryUntil('Wait for background css to be applied to first element', Step.sync(() => {
-      const p = SugarElement.fromDom(editor.getBody().firstChild);
+  const pWaitForVisualBlocks = (editor: Editor, waitUntilEnabled: boolean = true) =>
+    Waiter.pTryUntil('Wait for background css to be applied to first element', () => {
+      const p = TinyDom.fromDom(editor.getBody().firstChild);
       const background = Css.get(p, 'background-image');
-      Assertions.assertEq('Paragraph should have a url background', true, background.indexOf('url(') === 0);
-    }));
-  };
+      if (waitUntilEnabled) {
+        assert.include(background, 'url(', 'Paragraph should have a url background');
+        assert.isTrue(Class.has(TinyDom.body(editor), 'mce-visualblocks'), 'Visual blocks class should exist');
+      } else {
+        assert.notInclude(background, 'url(', 'Paragraph should not have a url background');
+        assert.isFalse(Class.has(TinyDom.body(editor), 'mce-visualblocks'), 'Visual blocks class should not exist');
+      }
+    });
 
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyApis = TinyApis(editor);
+  it('TBA: Toggle on/off visualblocks and compute previews', async () => {
+    const editor = hook.editor();
+    editor.execCommand('mceVisualBlocks');
+    await pWaitForVisualBlocks(editor);
+    assert.include(editor.formatter.getCssText('h1'), 'border:0px none', 'Should not have a border');
```

Comment:
This should be `notInclude` shouldn't it?

Reply:
I changed this test because the previous test would had the possibility of failing silently if the style was anything other than `border:1px dashed` (e.g. if the style was changed to something like `1.5px dashed` and this functionality was broken)

This failing in FF because the text is:

```
"font-family:-apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif;font-size:32px;font-weight:700;font-style:normal;text-decoration:rgb(0, 0, 0);text-transform:none;background-color:rgba(0, 0, 0, 0);border:;border-radius:;outline:;text-shadow:none;"
```

`border:;`

Thoughts?

Reply:
That wouldn't work if there was `border-width` used in the content CSS either. I think the best way would be to declare a custom `content_styles` in the init so we can do an assertion that it's never the value we've specified?